CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
-# Contributing
+#
```

Comment:
@mrlolethan Well, the important here is you alerted me about the typo. I'll edit the file and fix it. And yeah, sometimes the in-browser editor break lines by itself :(